swagger.yml

```diff
@@ -656,7 +656,55 @@ paths:
             status: 409
             message: "You cannot like a comment twice"
           
-
+  /profiles/followers:
+    get:
+      tags: 
+      - Profiles
+      summary: "User can view the users following him"
+      description: "This endpoint enables another user to be able to view the users following him or her"
+      responses:
+        200:
+          description: "Followers returned successfully"
+          example:
+            status: 200
+            message: "Followers returned successfully"
+            data:
+              - followee: 'Dimkpa'
+                followers:
+                 id: 'ab32abe7-8233-482e-a6fe-d4ffab90c9be'
+                 firstname: 'Joy'
+                 lastname: 'Adam'
+                 email: 'j.adam@example.com'
+        400:
+          description: "A user that is not authenticated makes request to view followers list"
+          example:
+            status: 401
+            message: "Error in verification. Please try again"
+  /profiles/followee:
```

Comment:
Very good job on this. Please fix your error codes and error messages in the swagger documentation.

Reply:
Alright, will make this change @TheDrizzyWay